pkg/storaged/btrfs/utils.jsx

```diff
@@ -75,3 +79,12 @@ export function parse_subvol_from_options(options) {
     else
         return null;
 }
+
+export function validate_subvolume_name(name) {
+    if (name === "")
+        return _("Name cannot be empty.");
+    if (name.length > 255)
+        return _("Name cannot be longer than 255 characters.");
+    if (name.includes('/'))
+        return cockpit.format(_("Name cannot contain the character '/'."));
```

Comment:
What about subvolumes in directories? Should we allow those to be created in Cockpit?

Reply:
By giving a name like "dir/dir/subvol" when creating a new one, maybe. Cockpit would do the necessary "mkdir -p".

But I agree that this might be confusing.

Reply:
Oh, btrfs itself has a -p option, neat!  But, yeah, no strong opinions here.